src/tests/test_utils.py

```diff
@@ -704,14 +707,30 @@ def establish_cis_connection(transport, central, peripheral, trace, params, acl_
     return success, cisConnectionHandles
 
 
+def calc_conn_supervision_timeout_ms(iso_interval_ms):
+    """
+    Calculate the minimum ACL supervision timeout that can be used to create a CIG with given ISO_Interval
+    :param iso_interval: ISO_Interval
+    :return: connSupervisionTimeout
+    """
+    # According to Core 5.3 | Vol 6, Part B 4.5.13.2 CIS events and subevents:
+    # "ISO_Interval (...) shall be less than half the connSupervisionTimeout for the associated ACL."
+    # Thus, if we want to create CIG with requested parameters, the connSupervisionTimeout shall be valid
+    supervision_timeout = max(Initiator.supervisionTimeoutDefault * 10, int(iso_interval_ms * 2))
```

Comment:
Why not drop the `max` and just something like `int(iso_interval_ms * 2 + 10)`? Using exactly `int(iso_interval_ms * 2)` would violate the specification, which uses less/greater than (not less-or-equal/greater-or-equal).

Reply:
I wanted to be sure that the supervision timeout will be at least Initiator.supervisionTimeoutDefault. It's because I've seen Packetcraft is disconnecting the link when supervision timeout is less than that value (I do not know what's the lowest working value of supervisionTimeout). My assumption was the 250ms was took arbitrary (because you've seen such problem before).

Please take a look:
> HCI Event: LE Meta Event (0x3e) plen 31                                                                                                               #2826 [hci10]
      LE Enhanced Connection Complete (0x0a)
        Status: Success (0x00)
        Handle: 0
        Role: Slave (0x01)
        Peer address type: Public (0x00)
        Peer address: 45:67:89:AB:CD:EF (OUI 45-67-89)
        Local resolvable private address: 00:00:00:00:00:00 (Non-Resolvable)
        Peer resolvable private address: 00:00:00:00:00:00 (Non-Resolvable)
        Connection interval: 30.00 msec (0x0018)
        Connection latency: 0 (0x0000)
        Supervision timeout: 170 msec (0x0011)
        Master clock accuracy: 0x07
(...)
> HCI Event: Command Complete (0x0e) plen 6                                                                                                             #2834 [hci10]
      LE Setup Isochronous Data Path (0x08|0x006e) ncmd 1
        Status: Success (0x00)
        Handle: 1
< ISO Data TX: Handle 1 flags 0x02 dlen 499                                                                                                             #2835 [hci10]
> HCI Event: Disconnect Complete (0x05) plen 4                                                                                                          #2836 [hci10]
        Status: Success (0x00)
        Handle: 0
        Reason: Connection Timeout (0x08)

I'll create an issue for it and put lower prio on it. I would like to proceed with the test cases (as I see this is bug in Packetcraft we can workaround in EDTT)